.docker/docker-tests.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+
+readonly ANSI_BOLD="\\033[1m"
+readonly ANSI_RED="\\033[31m"
+readonly ANSI_GREEN="\\033[32m"
+readonly ANSI_BLUE="\\033[34m"
+readonly ANSI_RESET="\\033[0m"
+
+function print_info() {
+	if [ -t 1 ]; then
+		printf "${ANSI_BOLD}${ANSI_BLUE}%s${ANSI_RESET}\\n" "$1"
+	else
+		echo "$1"
+	fi
+}
+
+function print_success() {
+	if [ -t 1 ]; then
+		printf "${ANSI_BOLD}${ANSI_GREEN}%s${ANSI_RESET}\\n" "$1"
+	else
+		echo "$1"
+	fi
+}
+
+function print_error() {
+	if [ -t 1 ]; then
+		printf "${ANSI_BOLD}${ANSI_RED}%s${ANSI_RESET}\\n" "$1" 1>&2
+	else
+		echo "$1" 1>&2
+	fi
+}
+
+function executeWithArguments() {
+	rm -rf stdout stderr logs out err
+	LAST_ARGUMENTS=$*
+	"$@" > out 2> err || true
+	if [ -f stdout ]; then
+		mv stdout out
+	else
+		touch out
+	fi
+	if [ -f stderr ]; then
+		mv stderr err
+	else
+		touch err
+	fi
+}
```

Comment:
@gcotelli this may be intended for windows, but docker tests don't run on windows so this wasn't a problem

> standardIOStreamNamed: moniker forWrite: forWrite
> 	"Create the requested stdio (#stdin, #stdout, #stderr) if possible.
> 	On Windows, if stdio is not available, create a regular file instead."